src/dymoprint/command_line.py

```diff
@@ -131,6 +151,10 @@ def parse_args():
     return parser.parse_args()
 
 
+def mm_to_payload(mm, margin):
+    return (mm * 7) - margin * 2
```

Comment:
Do you understand this formula? I think it's confusing and it'd be nice to add some sort of docstring.

I think maybe it's just 7 pixels / mm, and then subtracting off both left and right margins?

Reply:
Adding a docstring. This was just as magic numbers before, but now I had max and min length I refactored out to a function.